kpi/serializers/v1/asset.py

```diff
@@ -49,12 +49,37 @@ class Meta(AssetSerializerV2.Meta):
                   'permissions',
                   'settings',)
 
+    # Restricting permissions with `get_grant_permissions` add extra queries
+    # TODO optimize this
```

Comment:
Could we fix it by simply having `get_grant_permissions()` do the filtering in Python instead of calling `filter()`?

Reply:
![I feel good about this](https://user-images.githubusercontent.com/2085013/65299102-13e7f480-db3c-11e9-81d8-f4327588f1fb.gif)